src/pages/dar_application/collaborator/CollaboratorSummary.js

```diff
@@ -89,44 +96,14 @@ export const CollaboratorSummary = (props) => {
             }
           }),
         ]),
-      ]),
-      // Delete Confirmation Buttons
-      // Cancel Delete
-      div({
-        className: 'collaborator-summary-confirm-delete-buttons',
-        isRendered: props.deleteMode,
-      }, [
-        a({
-          id: index+'_editCollaborator',
-          style: { marginLeft: 10, marginRight: 10 },
-          onClick: () => props.toggleDeleteBool(false),
-        }, [
-          span({
-            className: 'glyphicon glyphicon-remove caret-margin collaborator-cancel-delete-icon', 'aria-hidden': 'true',
-            'data-tip': 'Edit dataset', 'data-for': 'tip_edit'
-          }),
-          span({
-            style: {
-              marginLeft: '1rem',
-            }
-          }),
-        ]),
-        // Confirm Delete
-        a({
-          id: index+'_confimDeleteMember',
-          style: { marginLeft: 10 },
-          onClick: () => props.deleteCollaborator(),
-        }, [
-          span({
-            className: 'glyphicon glyphicon-trash collaborator-confirm-delete-icon',
-            'aria-hidden': 'true', 'data-tip': 'Delete dataset', 'data-for': 'tip_delete'
-          }),
-          span({
-            style: {
-              marginLeft: '1rem',
-            }
-          }),
-        ]),
+        // Delete Confirmation Modal
+        h(ConfirmationModal, {
```

Comment:
I think this is the root of the issue with the modal display. This component is not as flexible as we need for this use case. I think it might be better to create a new modal specifically for this function. `ConfirmationModal` has a rather complicated history and tries to be all things for all cases.